vNext/test/TestExtensions/TestExtensions.csproj

```diff
@@ -48,6 +48,9 @@
     <Compile Include="..\..\..\test\TestExtensions\TestDefaultConfiguration.cs">
       <Link>TestDefaultConfiguration.cs</Link>
     </Compile>
+    <Compile Include="..\..\..\test\TestExtensions\TesterUtils.cs">
+      <Link>TesterUtils.cs</Link>
+    </Compile>
```

Comment:
In that case let's revert it (I also already reverted the move for the other TestUtils class) . Let's not clean these projects by polluting the clean ones with dependencies that once other code relies on it is harder to cleanup. Tester and TesterInternal projects' references are already very bad, but at least it's isolated to those, and we have a plan to eventually get rid of those projects, but not the clean ones.